docs/HowToUseMultiHeap.md

```diff
@@ -57,3 +58,12 @@ CONFIG_RAM_REGIONx_SIZE="1024,512,2048,512,256,4096"
 CONFIG_RAM_REGIONx_HEAP_INDEX="0,1,1,2,2,3"
 ```
 
+## 4. Supported APIs
+Following APIs can be used to allocate memory for a specific heap.
```

Comment:
1. Could you leave the header file name?
2. Could you leave descriptions for original APIs? When I use malloc on multi heap, how does it work against malloc_at?